front/commonitilvalidation.form.php

```diff
@@ -92,6 +92,40 @@
                 sprintf(__('%s adds an approval'), $_SESSION["glpiname"])
             );
         }
+    } else if ($_POST['validatortype'] == 'requester_supervisor') {
+        if (!isset($_POST['itemtype'])) {
+            Html::back();
+        }
+        $itemtype = $_POST['itemtype'];
+        $itemtypeFk = $itemtype::getForeignKeyField();
+        if (!isset($_POST[$itemtypeFk])) {
+            Html::back();
+        }
+
+        $itilObject = new $itemtype();
+        if (!$itilObject->getFromDB($_POST[$itemtypeFk])) {
+            Html::back();
+        }
+        $primaryRequester = $itilObject->getPrimaryRequesterUser();
+        if ($primaryRequester === null) {
+            Html::back();
+        }
+        if ($primaryRequester->fields['users_id_supervisor'] == 0) {
+            // TRANS: $1%s us the friendly user name
+            Session::addMessageAfterRedirect(sprintf(__('%1$s does not have a responsible'), $primaryRequester->getFriendlyName()));
```

Comment:
From what I understand, responsible can only be used as an adjective in English.
You can be "responsible for something" or "responsible of something" but you can't be "a responsible".

Reply:
An issue has been opened for this topic. As I said to Curtis, this PR is quite big, it is better to focus on the feature and fix this wording problem separately.